pkg/features/features.go

```diff
@@ -70,6 +70,9 @@ const (
 
 	// EnableQuotaAdmission enables quota admission.
 	EnableQuotaAdmission featuregate.Feature = "EnableQuotaAdmission"
+
+	// Enable DevicePlugin GPU shared resource
+	EnableDPGPUSharedResource featuregate.Feature = "EnableDPGPUSharedResource"
```

Comment:
Should be called EnableSyncGPUSharedResource？

Reply:
replace with EnableSyncGPUSharedResource